### PR TITLE
improved readability

### PR DESCRIPTION
**Pull request by alexr1997**

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/git/essentials/remote-repository/introduction-to-remote-repositories.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/git/essentials/remote-repository/introduction-to-remote-repositories.md?ref=2ae5644d8a086a276561621af317266d8596a289)

File: git/essentials/remote-repository/introduction-to-remote-repositories.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ links:
 
 You can have multiple remote repositories, with different permissions on each one. Some can be **read-only** while on some you can perform both **read** and **write** operations.
 
-These remote repositories are useful and even a requirement when collaborating as part of a team.
+These remote repositories are useful and sometimes even a requirement when collaborating as part of a team.
```
Comment (nem035) [inline git/essentials/remote-repository/introduction-to-remote-repositories.md]: More so "mostly" than "sometimes"

Comment (kapnobatai137) [inline git/essentials/remote-repository/introduction-to-remote-repositories.md]:
```suggestion
These remote repositories are useful and, most of the time, represent a requirement when collaborating as part of a team.
```

Comment (kapnobatai137) [reply git/essentials/remote-repository/introduction-to-remote-repositories.md]: what do you think of this @nem035?

Comment (nem035) [reply git/essentials/remote-repository/introduction-to-remote-repositories.md]: sure, that works

**Pull request merged**